extendr-api/src/wrapper/list.rs

```diff
@@ -60,7 +73,7 @@ impl IntoIterator for List {
     type IntoIter = NamedListIter;
     type Item = (&'static str, Robj);
 
-    /// Convert a List into an interator, consuming th epairlist.
+    /// Convert a List into an interator, consuming the pairlist.
```

Comment:
Did you mean "consuming the list"? A pairlist is different from a regular list. I think here we're dealing with regular lists.